### ISSUE TITLE
How to set policy (user flow)?

### ISSUE THREAD

**Issue by ankur198** [mvertopoulos/vue-msal#11]
it would be better to have options like

```javascript

tenantConfig = {
        tenant: "hellowrold.onmicrosoft.com",
        clientID: '{ your-client-id }',
        signInPolicy: "B2C_1_signin",
        signUpPolicy: "B2C_1_signup",
        redirectUri:"http://localhost:4200",
        b2cScopes:["https://hellowrold.onmicrosoft.com/access-api/user_impersonation"]
 };
```

**Comment by adomsodi** [comment]
+1, 
B2C user flows should be configurable and I would welcome more control over the sign-in request parameters.
Unfortunately, I have to fall back to native MSAL.

**Comment by ddlaat** [comment]
@adomsodi Configure it like this:

```javascript
auth = {
  clientId: 'd2e486b0-4e06-43e0-9aaf-62054c032cf7',
  tenantId: '<your-tenant>.onmicrosoft.com/B2C_1_signup_signin', // <-- policy id
  tenantName: '<your-tenant>.b2clogin.com/tfp', // <-- TFP
  validateAuthority: false
}
```

**Comment by mvertopoulos** [comment]
Hi!

This has now been fixed with commit https://github.com/mvertopoulos/vue-msal/commit/7bf71601108a0c5de160b75d81336623685640a9. <br>
It has also been released in npm version [3.1.0](https://www.npmjs.com/package/vue-msal/v/3.1.0).

This makes the `authority` option available for use instead of the `tenantId` and `tenantName` options. To avoid braking changes the previous options `tenantId` and `tenantName` are also available in this version but they will be **removed in next major version**. To prepare for this change, existing users that want to upgrade to next **major** version (4.x.x) will need to replace:

```js
{
  auth: {
    tenantName: '<YOUR_tenantName>',  // Default is 'login.microsoftonline.com'
    tenantId: '<YOUR_tenantId>'  // Default is 'common'
    //...
  }
}
```

with:

```js
{
  auth: {
    authority: 'https://{tenantName}/{tenantId}' // Default is 'https://login.microsoftonline.com/common'
    //...
  }
}
```

Thanks for all of your inputs.
